pygeo/DVConstraints.py

```diff
@@ -5711,8 +5712,9 @@ def __init__(self, name, surfs, curvatureType, lower, upper, scaled, scale, KSCo
         for iSurf in range(self.nSurfs):
             self.curvatureRef += self.evalCurvArea(iSurf)[0]
 
-        if MPI.COMM_WORLD.rank == 0:
-            print("Reference curvature: ", self.curvatureRef)
+        # from mpi4py import MPI
+        # if MPI.COMM_WORLD.rank == 0:
+        #     print("Reference curvature: ", self.curvatureRef)
```

Comment:
I think the print should remain... but this could become messy when we use pygeo within an MPI job. We could use `pp` from [baseclasses](https://github.com/mdolab/baseclasses/blob/4697ae6154906f0c1f7d28dfd30319ead9adfd05/baseclasses/BaseSolver.py#L227-L241) but this would add the dependency from `baseclasses`... Opinions? @nwu63

Reply:
No we shouldn't use baseclasses because that requires rewriting the whole package to use BaseSolver which doesn't make sense to me.

Reply:
yeah nevermind,I was thinking about a direct import but that would be bad practice AND would probably get other MPI issues.